src/model/engines/engine_base.nit

```diff
@@ -80,6 +71,15 @@ class Engine
 		submission.update_status(config)
 	end
 
+	# Mark the `submission` as an error, and save it in the `config`
+	fun set_compilation_error(config: AppConfig, submission: Submission)
+	do
```

Comment:
You see me in the obligation to refer to the [Holly Rules of the Repository, chapter 3, section 1, alinea 2](https://github.com/Morriar/Missions/blob/master/README.md#code-style).

I cite:

> **The do is on the same line as the function declaration.**

Please, just stop this non-sense... The war is over, you just don't want to admit it.